docs/src/roadmap.md

```diff
@@ -11,14 +11,14 @@ JuMP 1.0 will be ready to release roughly when all of these tasks are completed.
 Some but not all of these tasks are summarized in the
 [JuMP 1.0 milestone](https://github.com/jump-dev/JuMP.jl/milestone/12).
 
-- Create a website for JuMP.
+- Create a website for JuMP. (**Done**: https://jump.dev)
```

Comment:
The website isn't done. It's still missing content from juliaopt.org like https://www.juliaopt.org/packages/.